packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/go/integrationsteps.md

```diff
@@ -118,11 +88,11 @@ if lr.HasStep(idx.LoginStepEmailVerification) {
 }
 ```
 
-### 6: Submit the email factor to verify the user's identity
+Use the other [LoginStep](https://github.com/okta/okta-idx-golang/blob/master/identify.go#L692)s to populate the rest of the list.
 
-When the user selects the email factor and clicks submit, call the `LoginResponse` object's
-`VerifyEmail` method. Calling this method instructs the Okta org server to send an
-email to the email address the user provided in [Step 3](#_3-submit-the-credentials).
+### 4: The user submits the email authenticator
+
+When the user submits the email authenticator, call `LoginResponse.VerifyEmail()`. Identity Engine sends a verification email to the user if the call is succcessful.
```

Comment:
```suggestion
When the user submits the email authenticator, call `LoginResponse.VerifyEmail()`. Identity Engine sends a verification email to the user if the call is succcessfull.
```

Reply:
Not changing. Successful is spelt with only one l. Even in American English.